commands/python/libs/k8s/helm.py

```diff
@@ -131,23 +155,36 @@ def release_exists(self, release_name):
         else:
             return False
 
-    def rollback_relative(self, helm_rules, num_versions):
+    def rollback_relative(self, helm_rules, num_versions, namespace):
         release_name = helm_rules.release_name
 
         # Some ugly logic to get the current revision - probably can be better
-        output = subprocess.Popen(("helm", "list"), stdout=subprocess.PIPE)
+        helm_list_command = [self.helm, "list"]
+        # Helm3 requires --namespace flag, which helm2 doesn't use
+        if self.helm == "helm":
+            helm_list_command.extend(["--namespace", namespace])
+        output = subprocess.Popen(helm_list_command, stdout=subprocess.PIPE)
         output = subprocess.check_output(("grep", release_name), stdin=output.stdout)
-        current_revision = int(output.decode("utf-8").replace(" ", "").split("\t")[1])
+        data = output.decode("utf-8").replace(" ", "").split("\t")
```

Comment:
nit: this seems to be working fine but would it be better to add a "--output json|yaml" flag on the command to explicitly grab the revision versus selecting a column below?

Reply:
Good suggestion - I will take that on in the forthcoming PR which will remove helm2 completely if that works